### PR TITLE
fixup! IKASAN-1669 - Improve performance and simplicity of the EventG…

### PR DESCRIPTION
**Pull request by mitcje**
…eneratingConsumer.

### CODE REVIEW

File: ikasaneip/component/endpoint/test-endpoint/Readme.md [hunk 1]
```diff
@@ -1,17 +1,118 @@
...
+###### Sample Usage - Ikasan Java FluentAPI
+Example below shows how easy it is to provide your own custom event endpoint provider.
+
+```java
+public class ModuleConfig 
+{
+  @Resource
+  private BuilderFactory builderFactory;
+
+  public Consumer getScheduledConsumer()
```
Comment (andrzej-majewski) [inline ikasaneip/component/endpoint/test-endpoint/Readme.md]: this method should be getEventGeneratingConsumer

File: ikasaneip/component/endpoint/test-endpoint/Readme.md [hunk 2]
```diff
@@ -1,17 +1,118 @@
...
+You can optionally set the ManagedEventIdentifierService specific to this event provider if the default which simply creates a hashcode of the entire event is not sufficient.
+
+```java
+public class ModuleConfig 
+{
+  @Resource
+  private BuilderFactory builderFactory;
 
-##### Sample Usage
+  public Consumer getScheduledConsumer()
```
Comment (andrzej-majewski) [inline ikasaneip/component/endpoint/test-endpoint/Readme.md]: this method should be getEventGeneratingConsumer

Comment (mitcje) [reply ikasaneip/component/endpoint/test-endpoint/Readme.md]: Fixed

File: ikasaneip/component/endpoint/test-endpoint/Readme.md (hunk 1 above)
Comment (mitcje) [reply ikasaneip/component/endpoint/test-endpoint/Readme.md]: getEventGeneratingConsumer

**Pull request merged**